ulauncher/modes/extensions/extension_remote.py

```diff
@@ -86,6 +87,7 @@ def _get_download_url(self, commit: str) -> str:
 
     def _get_refs(self) -> dict[str, str]:
         refs = {}
+        ref = None
```

Comment:
This looks unused?

Reply:
This is to fix the "could be unassigned variable" linter error

Reply:
It was Pyright. It's probably better to keep this change. If the `try` block fails, the `except` block will try to reference an undeclared variable.